.github/workflows/badgerdoc-storage.yml

```diff
@@ -0,0 +1,32 @@
+name: badgerdoc_storage
```

Comment:
I want to rename all CI GH Actions to match the microservice/library name to avoid job names like the ones in the example:
![Screenshot 2024-09-02 at 22 29 38](https://github.com/user-attachments/assets/28503174-bfb1-4747-971c-9b4d6208e5db)

Example: 
`badgerdoc_storage`, `users`, `assets`, etc. It's much easier to understand which CI is failed.